daemons/fenced/fence_watchdog.in

```diff
@@ -0,0 +1,278 @@
+#!@PYTHON@
+"""Dummy watchdog fence agent for providing meta-data for the pacemaker internal agent
+"""
+
+# Pacemaker targets compatibility with Python 2.7+ and 3.2+
+from __future__ import print_function, unicode_literals, absolute_import, division
```

Comment:
The whole dummy-agent is untouched since I left it alone 1 1/2 years ago ;-)
Probably requires some deeper inspection but as it seemed to do it's job
for now I first went for the other stuff since it is quite clear what this one
should do.